ui/de-lib/src/main/resources/org/iplantc/de/resources/client/messages/IplantContextualHelpStrings.properties

```diff
@@ -7,3 +7,4 @@ toolRequestStatusInstallationHelp = The iPlant team is currently installing your
 toolRequestStatusValidationHelp = The iPlant team is currently validating your requested tool. You will receive notifications in the Discovery Environment as the process proceeds. Please see the Status Comments for more information.
 toolRequestStatusCompleteHelp = Your requested tool is now available in the Discovery Environment. Please see the Status Comments for more information.
 toolRequestStatusFailedHelp = The installation of your requested tool was unsuccessful. Please see the Status Comments for more information.
+toolRequestStatusOtherHelp = Please see the Status Comments for more information.
```

Comment:
Gotcha - for some reason I thought they were the same file.

I found the setting in IntelliJ.  It was checked already, so I unchecked, then rechecked.  Seems to be working now.